tabbycat/tournaments/views.py

```diff
@@ -157,6 +159,21 @@ def post(self, request, *args, **kwargs):
             return super().post(request, *args, **kwargs)
 
 
+class SendStandingsEmailsView(RoundMixin, AdministratorMixin, PostOnlyRedirectView):
+
+    def post(self, request, *args, **kwargs):
+        active_teams = Team.objects.filter(debateteam__debate__round=self.round)
+
```

Comment:
This isn't obvious, and I should really add a docstring about this, but `Team.points_count` is actually query-heavy if it's not prefetched—it'll generate a SQL query per team. There's a function for this purpose called `populate_win_counts` in participants/prefetch.py, so if you add to the imports
``` python
from participants.prefetch import populate_win_counts
```
and then after line 165, add
``` python
populate_win_counts(active_teams)
```
then it'll save a lot of SQL queries when sending this emails.

Reply:
Commit to avoid repeating this I see: 38c6473 :)